bazel/com_google_cel_cpp.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/eval/public/cel_function_adapter.h b/eval/public/cel_function_adapter.h
+index d99239c..4dc8cae 100644
+--- a/eval/public/cel_function_adapter.h
++++ b/eval/public/cel_function_adapter.h
+@@ -118,6 +118,34 @@ class FunctionAdapter : public CelFunction {
+     return registry->Register(std::move(status.ValueOrDie()));
+   }
+ 
++#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
Can we do
`#if defined(__clang_major_version__) && __clang_major_version__ >= 8 && !defined(__APPLE__)` for optimized version, and for the rest use the fallback, (possibly with #warning macro message)?

Reply:
Sure.